web/react/components/posts_view.jsx

```diff
@@ -38,6 +40,15 @@ export default class PostsView extends React.Component {
     static get SCROLL_TYPE_POST() {
         return 5;
     }
+    forceUpdateInfo() {
```

Comment:
Without it I was only getting the names to change if I refreshed the page (or loaded into a channel I hadn't viewed recently, but going back to the other channel had the names as they were).  It looked kind of broken since you were changing the setting and then not seeing the results immediately.  If it's not worth the performance hit or if there's a better way to do it let me know.